indexer/feature_data.cpp

```diff
@@ -98,9 +98,9 @@ class UselessTypesChecker
   {
     // 2-arity is better than 1-arity
 
-    ftype::TruncValue(t, 2);
-    if (IsIn2(t))
-      return 1;
+//    ftype::TruncValue(t, 2);
+//    if (IsIn2(t))
```

Comment:
Why leave it commented when everything else (e.g. `IsIn2()`) is deleted?

Reply:
Also the comment just above is not needed anymore.

Reply:
I prefer to keep it like self-documented, how it should work if will be needed to restore.